README.md

```diff
@@ -54,6 +54,8 @@ In LoopBack applications, add the following entry to your
   "final:after": {
     "strong-error-handler": {
       "params": {
+         "debug": true,
```

Comment:
By default, loopback-workspace scaffolds LoopBack applications with `debug: false` for security reasons. I think we should use `debug: false` here too, and add `server/middleware.development.json` with `debug:false`.

I see that we are using `debug:true`on L44 where we are showing usage in express apps. I think that place should be changed to `debug: app.get('env') === 'development'` or similar (I am not sure if my code snippet here works).